pro-client/enable_realtime_kernel.rst

```diff
@@ -1,92 +1,146 @@
     `services compatibility matrix. <https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/references/compatibility_matrix/>`_.
+    
+    If you try to enable Real-time Ubuntu while an incompatible service is
+    already enabled, the Pro Client will inform you and offer to disable that
+    service.
 
-To ``enable`` the real-time kernel through the Ubuntu Pro Client, please run:
+Enable and install automatically
+================================
 
-.. code-block:: bash
+The Real-time Ubuntu kernel package is installed using the APT package manager
+when you enable ``realtime-kernel`` through the Pro Client.
 
-    $ sudo pro enable realtime-kernel
+If you want to access the repository but not install the package immediately,
+skip to `Install and enable manually`_.
 
-You'll need to acknowledge a warning, and then you should see output like the
-following, indicating that the real-time kernel package has been installed.
+Otherwise, `select the correct version <rt_releases_>`_ for your OS
+and processor, and use the corresponding command below:
 
-.. code-block:: text
+.. tab-set::
 
-    One moment, checking your subscription first
-    The Real-time kernel is a beta version of the 22.04 Ubuntu kernel with the
-    PREEMPT_RT patchset integrated for x86_64 and ARM64.
+   .. tab-item:: Generic
 
-    This will change your kernel. You will need to manually configure grub to
-    revert back to your original kernel after enabling real-time.
+      .. code-block:: shell
 
-    Do you want to continue? [ default = Yes ]: (Y/n) yes
-    Updating package lists
-    Installing Real-time kernel packages
-    Real-time kernel enabled
-    A reboot is required to complete install.
+         sudo pro enable realtime-kernel
+      
+      .. caution::
+         The generic Real-time kernel is not intended for Raspberry Pi.
+         Using the Pro Client to install it on these platforms will make your
+         system unusable.
 
-After rebooting you'll be running the real-time kernel!
+   .. tab-item:: Raspberry Pi
 
-Enable and manually install
-===========================
+      For Raspberry Pi 4 and 5:
 
-.. important::
+      .. code-block:: shell
 
-    The ``--access-only`` flag was introduced in Pro Client version 27.11
+         sudo pro enable realtime-kernel --variant=raspi
 
-If you would like to enable access to the real-time kernel APT repository but
-not install the kernel right away, use the ``--access-only`` flag when you
-enable it, as follows:
+   .. tab-item:: Intel IOTG
 
-.. code-block:: bash
+      For 12th Gen Intel® Core™ processors:
 
-    $ sudo pro enable realtime-kernel --access-only
+      .. code-block:: shell
 
-With this extra flag you'll see output like this:
+         sudo pro enable realtime-kernel --variant=intel-iotg
+
+You will need to acknowledge a warning, then you will see a confirmation
+message like the following that the Real-time Ubuntu package has been
+installed:
 
 .. code-block:: text
```

Comment:
Formatting should be italics here in this code block? Similar code from line 116 is italicised?

Reply:
I'm not sure what's the source of this exactly. I've made some changes to the base document in the Pro Client repo which I can port over once they're merged there - if this doesn't fix it, I'll need to change the CSS styling.